lib/iris/tests/integration/test_netcdf.py

```diff
@@ -273,6 +274,100 @@ def test_print(self):
                          '    x         x') in printed)
 
 
+class TestCMIP6VolcelloLoad(tests.IrisTest):
```

Comment:
These tests need to be qualified with `@tests.skip_data`
So, any tests that get test data (via tests.get_data_path) *must* have `@skip_data'.

This is why the 'TEST_MINIMAL' entries in the Travis matrix are failing
 - which should never happen, as 'minimal' tests are a strict subset of all tests
 - the idea is basically just that the 'minimal' tests run quicker (and part of that is not installing iris-test-data)

Reply:
smart, fixed now